File: Two Sum II - Input Array Is Sorted.py
class Solution:
    def twoSum(self, arr: List[int], target: int) -> List[int]:
        left=0
        right=len(arr)-1
        while left<right:
            summ=arr[left]+arr[right]
            if summ>target:
                right-=1
            elif summ==target:
                return [left+1,right+1]
            else:
                left+=1

        return ""

File: Java/Two Sum II - Input Array Is Sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int left = 0 ;
        int right =numbers.length-1;
        int total=0;
        while( left<right ){
            total=numbers[left]+numbers[right];
            if(total==target){
                return new int[]{left+1,right+1};
            }
            if ( total>target){
                right--;
            }
            else{
                left++;
            }
            
        }
        return new int[]{};
    }
}
